ops/terraform/modules/stateless/main.tf

```diff
@@ -352,7 +352,7 @@ module "bfd_server_alarm_all_eob_6s-p95" {
     datapoints       = "15"
     statistic        = null
     ext_statistic    = "p95"
-    threshold        = "6000.0"
+    threshold        = "6000000.0"
```

Comment:
Microseconds are an egregious violation of the Principle of Least Surprise. Accordingly, this line (and any similar lines in our Terraform) really needs a comment listing the unit.